tools/src/main/java/org/apache/kafka/trogdor/fault/NoOpFaultSpec.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.trogdor.fault;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import org.apache.kafka.trogdor.common.JsonUtil;
+
+import java.util.Objects;
+
+/**
+ * The specification for a fault that does nothing.
+ *
+ * This fault type exists mainly to test the fault injection system.
+ */
+public class NoOpFaultSpec implements FaultSpec {
+    private final long startMs;
+    private final long durationMs;
```

Comment:
Perhaps there could be an `AbstractFaultSpec` that captures common code? Otherwise, every new `FaultSpec` implementation will be adding code the size of this class.